### PR TITLE
Typescript Story: Avatar

### PR DESCRIPTION
**Pull request by pomfrida**
Part of #735

### CODE REVIEW

File: apps/storybook-react/stories/Avatar.stories.tsx [hunk 1]
```diff
@@ -20,10 +20,13 @@ const Wrapper = styled.div`
 export default {
   title: 'Components/Avatar',
   component: Avatar,
-  decorators: [withKnobs],
-}
+} as Meta
 
-export const Examples = () => (
+export const Default: Story<AvatarProps> = (args) => (
+  <Avatar src={image} {...args}></Avatar>
```
Comment (vnys) [inline apps/storybook-react/stories/Avatar.stories.tsx:26]: Shouldn’t this be self-closing?

```suggestion
  <Avatar src={image} {...args} />
```

**Review by vnys — COMMENTED** [review]
I think the `<Avatar />` component should be self closing, but apart from that it LGTM 👍

File: apps/storybook-react/stories/Avatar.stories.tsx (hunk 1 above)
Comment (pomfrida) [reply apps/storybook-react/stories/Avatar.stories.tsx:26]: Yes! I'll fix!

**Review by wenche — APPROVED** [review]
LGTM 👍

**Comment by pomfrida** [comment]
Changed to **WIP** again due to this module 

```
declare module '*.jpg' {
  const value: string
  export = value
}
```

Needed for the image import in `Avatar.stories.tsx`. This module is currently only reachable from the story when the `custom.d.ts` file is open locally, that's the only way the import error is cleared.

I've tried adding `compilerOptions: typeRoots` in `.storybook/main.js`,` tsconfig.base.json` (root) and in core-react `tsconfig` without any luck. 

Check out Stackoverflow link about [TS image import](https://stackoverflow.com/questions/51100401/typescript-image-import)

**Comment by mimarz** [comment]
> Changed to **WIP** again due to this module
> 
> ```
> declare module '*.jpg' {
>   const value: string
>   export = value
> }
> ```
> 
> Needed for the image import in `Avatar.stories.tsx`. This module is currently only reachable from the story when the `custom.d.ts` file is open locally, that's the only way the import error is cleared.
> 
> I've tried adding `compilerOptions: typeRoots` in `.storybook/main.js`,` tsconfig.base.json` (root) and in core-react `tsconfig` without any luck.
> 
> Check out Stackoverflow link about [TS image import](https://stackoverflow.com/questions/51100401/typescript-image-import)

Suggest just screwing it and use this link for the 😺 : https://i.imgur.com/UM3mrju.jpg
We have better things to work on 🙈

**Comment by pomfrida** [comment]
Did the quick fix @mimarz suggested to avoid TS error on image import.

**Pull request merged**